### ISSUE TITLE
iPad is unable to connect

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#3506]
The symptom is:

* Connects, gets to about two seconds of “connected”
* Then goes to unstable connection
* Then never connects

Can DM logs to whoever needs it.

`Failed to receive packet: address family not supported by protocol family`

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-2208)
┆author: Sarah Bird

**Comment by data-sync-user** [comment]
➤ Sarah Bird commented:

An update. I uninstalled and reinstalled then the following wierd things happened.

  1. It said it connected and looked like everything was fine - the connection timer kept ticking up
  2. But then I tried to use the internet and the internet was down.
  3. So I went back, disconnected and reconnected and it DID connect - for real - I checked the mullvad home page

Here’s some log snippets. I can share on dm if needed:

5/12/22, 3:54:29 PM Central Daylight Time - Routine: decryption worker - started
5/12/22, 3:54:29 PM Central Daylight Time - Failed to receive packet: address family not supported by protocol family
5/12/22, 3:54:30 PM Central Daylight Time - Failed to receive packet: address family not supported by protocol family
5/12/22, 3:54:30 PM Central Daylight Time - Failed to receive packet: address family not supported by protocol family
5/12/22, 3:54:30 PM Central Daylight Time - Network change detected with satisfied route and interface order [en0, utun3, pdp_ip0]
5/12/22, 3:54:30 PM Central Daylight Time - DNS64: mapped 37.19.210.27 to itself.
5/12/22, 3:54:30 PM Central Daylight Time - peer(9BP/‚Ä¶kO2k) - UAPI: Updating endpoint
5/12/22, 3:54:30 PM Central Daylight Time - Failed to receive packet: address family not supported by protocol family
5/12/22, 3:54:31 PM Central Daylight Time - Failed to receive packet: address family not supported by protocol family
5/12/22, 3:54:31 PM Central Daylight Time - Failed to receive packet: address family not supported by protocol family
5/12/22, 3:54:31 PM Central Daylight Time - Failed to receive packet: address family not supported by protocol family
5/12/22, 3:54:32 PM Central Daylight Time - Failed to receive packet: address family not supported by protocol family
5/12/22, 3:54:32 PM Central Daylight Time - Failed to receive packet: address family not supported by protocol family
5/12/22, 3:54:32 PM Central Daylight Time - Failed to receive packet: address family not supported by protocol family
5/12/22, 3:54:33 PM Central Daylight Time - Failed to receive packet: address family not supported by protocol family
5/12/22, 3:54:33 PM Central Daylight Time - Routine: receive incoming IPv6 - stopped
5/12/22, 3:54:33 PM Central Daylight Time - UDP bind has been updated
5/12/22, 3:54:33 PM Central Daylight Time - Routine: receive incoming IPv6 - started
5/12/22, 3:54:38 PM Central Daylight Time - peer(9BP/‚Ä¶kO2k) - Handshake did not complete after 5 seconds, retrying (try 2)
5/12/22, 3:54:38 PM Central Daylight Time - peer(9BP/‚Ä¶kO2k) - Sending handshake initiation
5/12/22, 3:56:36 PM Central Daylight Time - Stopping tunnel
5/12/22, 3:56:36 PM Central Daylight Time - Device closing
5/12/22, 3:56:36 PM Central Daylight Time - Routine: event worker - stopped
5/12/22, 3:57:02 PM Central Daylight Time - App version: 2.8.1 (2.202204262020); Go backend version: 0.0.0
5/12/22, 3:57:02 PM Central Daylight Time - Starting tunnel from the OS directly, rather than the app
5/12/22, 3:57:02 PM Central Daylight Time - DNS64: mapped 37.19.210.14 to itself.
5/12/22, 3:57:02 PM Central Daylight Time - Attaching to interface
5/12/22, 3:57:02 PM Central Daylight Time - UAPI: Updating private key
5/12/22, 3:57:02 PM Central Daylight Time - UAPI: Removing all peers
5/12/22, 3:57:02 PM Central Daylight Time - peer(8Hqj‚Ä¶ElBk) - UAPI: Created
5/12/22, 3:57:02 PM Central Daylight Time - Routine: encryption worker - started
5/12/22, 3:57:02 PM Central Daylight Time - Routine: decryption worker - started
5/12/22, 3:57:02 PM Central Daylight Time - peer(8Hqj‚Ä¶ElBk) - UAPI: Updating endpoint
5/12/22, 3:57:02 PM Central Daylight Time - peer(8Hqj‚Ä¶ElBk) - UAPI: Updating persistent keepalive interval
5/12/22, 3:57:02 PM Central Daylight Time - peer(8Hqj‚Ä¶ElBk) - UAPI: Removing all allowedips
5/12/22, 3:57:02 PM Central Daylight Time - Routine: decryption worker - started
5/12/22, 3:57:02 PM Central Daylight Time - Routine: encryption worker - started
5/12/22, 3:57:02 PM Central Daylight Time - peer(8Hqj‚Ä¶ElBk) - UAPI: Adding allowedip
5/12/22, 3:57:02 PM Central Daylight Time - Routine: decryption worker - started
5/12/22, 3:57:02 PM Central Daylight Time - peer(8Hqj‚Ä¶ElBk) - UAPI: Adding allowedip
5/12/22, 3:57:02 PM Central Daylight Time - Routine: handshake worker - started
5/12/22, 3:57:02 PM Central Daylight Time - Routine: encryption worker - started
5/12/22, 3:57:02 PM Central Daylight Time - Routine: handshake worker - startedNotice there’s more of the “failed to receive packet” but then after re-opening the app i don’t see that error any more.

**Comment by data-sync-user** [comment]
➤ Sarah Bird commented:

BUT…more drama….then I went back into the app and I’m afraid I can’t remember if I tried switching servers or not, but it went into “connecting” state. I let it sit there for a while and I finally got the “server unavailable” modal. This was for Seattle.

I picked Los Angeles and it’s working again……

Again…I have logs….but still not seeing {{Failed to receive packet: address family not supported by protocol family }} during the time I’m able to successfully connect.
